lib/execution_engine2/db/models/transfer/rollback_db.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python
+import os
+from datetime import datetime
+from bson.objectid import ObjectId
+import copy
+from configparser import ConfigParser
+from pymongo import MongoClient
+
+
+class RollbakDatabases:
+
+    def _get_ujs_connection(self):
+        parser = ConfigParser()
+        parser.read(os.environ.get("KB_DEPLOYMENT_CONFIG"))
+        ujs_host = parser.get("NarrativeJobService", "ujs-mongodb-host")
+        self.ujs_db = parser.get("NarrativeJobService", "ujs-mongodb-database")
+        ujs_user = parser.get("NarrativeJobService", "ujs-mongodb-user")
+        ujs_pwd = parser.get("NarrativeJobService", "ujs-mongodb-pwd")
+
+        return MongoClient(
+            ujs_host,
+            27017,
+            username=ujs_user,
+            password=ujs_pwd,
+            authSource=self.ujs_db,
+            retryWrites=False,
+        )
+
+    def _get_njs_connection(self):
+        parser = ConfigParser()
+        parser.read(os.environ.get("KB_DEPLOYMENT_CONFIG"))
+        njs_host = parser.get("NarrativeJobService", "mongodb-host")
+        self.njs_db = parser.get("NarrativeJobService", "mongodb-database")
+        njs_user = parser.get("NarrativeJobService", "mongodb-user")
+        njs_pwd = parser.get("NarrativeJobService", "mongodb-pwd")
+
+        return MongoClient(
+            njs_host,
+            27017,
+            username=njs_user,
+            password=njs_pwd,
+            authSource=self.njs_db,
+            retryWrites=False,
+        )
+
+    def _get_ee2_connection(self):
+        parser = ConfigParser()
+        parser.read(os.environ.get("KB_DEPLOYMENT_CONFIG"))
+        ee2_host = parser.get("NarrativeJobService", "mongodb-host")
+        self.ee2_db = "exec_engine2"
+        ee2_user = parser.get("NarrativeJobService", "mongodb-user")
+        ee2_pwd = parser.get("NarrativeJobService", "mongodb-pwd")
+
+        return MongoClient(
+            ee2_host,
+            27017,
+            username=ee2_user,
+            password=ee2_pwd,
+            authSource=self.ee2_db,
+        )
+
+    @classmethod
+    def _create_ujs_job_rec(self, ee2_job):
+
+        job_id = ee2_job["_id"]
+        job_input = ee2_job.get("job_input", {})
+        error = ee2_job.get("error", {})
+
+        ujs_job_doc = {"_id": job_id,
+                       "user": ee2_job.get("user"),
+                       "authstrat": ee2_job.get("authstrat"),
+                       "authparam": ee2_job.get("authstrat"),
+                       "created": job_id.generation_time,
+                       "updated": datetime.fromtimestamp(ee2_job.get("updated")),
+                       "estcompl": None,
+                       "service": ee2_job.get("user"),
+                       "complete": ee2_job.get("status") in ["completed", "terminated", "error"],
```

Comment:
it would be interesting to see how many jobs are not complete vs complete at the end of insertion.